src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -606,6 +606,7 @@ namespace NServiceBus
         void Customize(NServiceBus.EndpointConfiguration configuration);
     }
     public class InMemoryPersistence : NServiceBus.Persistence.PersistenceDefinition { }
+    public interface INotification { }
```

Comment:
Had to add it to get the generic's to play ball. I'll see if I can make it internal for now